.github/workflows/release.yml

```diff
@@ -4,7 +4,7 @@ on:
     inputs:
       formula:
         description: 'Formula to release Homebrew'
-        default: twilio
+        default: twilio@rc
```

Comment:
Does it not pass on the inputs from twilio-cli?

Reply:
It does not even ask for this in cli-core. We'll need to changed that

Reply:
[This](https://github.com/twilio/twilio-cli/blob/9bd5494264600ace8bd27752a1da59fecf53eda9/.github/workflows/oclif-release.yml#L69) line

Reply:
Oh, got that. Thanks! Approved.

Reply:
I Will change it after the release cause I don't want to spend time on testing the whole pipeline now